src/reports/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+set(reportinglib_SOURCES
+        "library/cell.cpp"
+        "library/cell.hpp"
```

Comment:
do the `.hpp` files contain sources, or are they just headers?
Seemingly, the convention in the code is headers are `.h`, and `.hpp` files are included into `.cpp` files, but not compiled.